.github/workflows/app_release.yml

```diff
@@ -0,0 +1,72 @@
+on:
```

Comment:
so this workflow can be triggered directly alone?

Reply:
it's reusable workflow, wouldn't be triggered manually but called by other workflows

Reply:
will this be listed though?
maybe we should move it to .github/workflows/templates/ to make it more obvious

Reply:
actually it cannot be 
`invalid value workflow reference: workflows must be defined at the top level of the .github/workflows/ directory`